sdk/data_kusto/src/operations/query.rs

```diff
@@ -0,0 +1,174 @@
+use crate::client::KustoClient;
+use crate::request_options::*;
+use azure_core::prelude::*;
+use azure_core::setters;
+use azure_core::{
+    collect_pinned_stream,
+    headers::{add_mandatory_header2, add_optional_header2},
+    Response as HttpResponse,
+};
+use futures::future::BoxFuture;
+
+/// A future of a delete file response
```

Comment:
I don't get it, when are you deleting a file?

Reply:
copy and paste error - will clean this up :)

Reply:
gone :)